src/assets/Components/Sidebar/Sidebar.css

```diff
@@ -0,0 +1,43 @@
+.sideBar {
+  height: 100%;
+  width: 50px;
```

Comment:
As you mentioned, it is really super small 😅
At least the sidebar works perfectly